### DISCUSSION TITLE
UuidType is not using uuid type in postgres

### DISCUSSION THREAD

**Discussion by xeoneux** [mikro-orm/mikro-orm#2906] [Q&A] [answered]
**Describe the bug**
When using postgres driver with an entity like
```
@PrimaryKey({ type: UuidType})
uuid!: UuidType
```
The generated migration has the type as varchar(255) instead of uuid

**Comment by B4nan** [comment, marked as answer]
It _does_ use the `uuid` type. 

https://github.com/mikro-orm/mikro-orm/blob/master/packages/core/src/types/UuidType.ts#L8
https://github.com/mikro-orm/mikro-orm/blob/master/packages/postgresql/src/PostgreSqlPlatform.ts#L72

Please dont ignore issue template if you want to open issues. What versions are you on? You probably have multiple `core` packages installed, which should be validate in v5.

**Comment by xeoneux** [comment]
I'm sorry, I figured out why this was happening. I've been using compiled js files with the cli instead of the ts files with new code!

Thanks for a quick reply @B4nan!
